common/server_protocolhandler.cpp

```diff
@@ -184,6 +184,18 @@ Response::ResponseCode Server_ProtocolHandler::processRoomCommandContainer(const
 
 Response::ResponseCode Server_ProtocolHandler::processGameCommandContainer(const CommandContainer &cont, ResponseContainer &rc)
 {
+    static QList<GameCommand::GameCommandType> antifloodCommandsWhiteList = QList<GameCommand::GameCommandType>()
```

Comment:
IIRC, a static variable in a class method will be allocated on the class and initialized only one, the first time the method gets executed.
I wrote a quick test for this, and it seems it's working as i expected: https://gist.github.com/ctrlaltca/7561aaa1ed6fae2e368e

Anyway, the reason why i didn't declare this variable as a class member in the .h is that it would force to add an ```#include "pb/commands.pb.h"``` in the .h, so every time a command gets modified it would force a recompilation of all the files including server_protocolhandler.h and so on.